### PR TITLE
README Improvements

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <maryjinglewski@honeycomb.io>

## Which problem is this PR solving?

N/A

## Short description of the changes

This PR reviews the Readme content and applies any copyediting or improvements for:
- [x] Repo README (overview)
- [x] [CloudWatch Logs](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/cloudwatch-logs)
- [x] [CloudWatch Metrics](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/cloudwatch-metrics)
- [x] [RDS Logs](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/rds-logs)
- [x] [Logs from a S3 Bucket](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/s3-logfile)

## How to verify that this has the expected result

Seeing is believing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,28 @@
...
-
-
 * [RDS Logs](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/rds-logs)
-
-
 * [Logs from a S3 Bucket](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/s3-logfile)
 
-## Use
+To install individual integrations, refer to each integration's README.
+Otherwise, refer to the [minimal configuration](#usage) to turn on all supported integrations.
```
Comment (mjingle) [inline README.md:21]: I know this needs work.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,28 @@
...
 * [CloudWatch Metrics](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/cloudwatch-metrics)
-
-
 * [RDS Logs](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/rds-logs)
-
-
 * [Logs from a S3 Bucket](https://github.com/honeycombio/terraform-aws-integrations/tree/main/modules/s3-logfile)
 
-## Use
+To install individual integrations, refer to each integration's README.
```
Comment (mjingle) [inline README.md:20]: Maybe a better word choice?
```suggestion
To configure individual integrations, refer to each integration's README.
```

File: README.md [hunk 3]
```diff
@@ -1,33 +1,28 @@
...
+Otherwise, refer to the [minimal configuration](#usage) to turn on all supported integrations.
+
+## Usage
 
-The minimal config to turn all integrations on is:
+The minimal Terraform configuration to turn on all [supported integrations](#supported-integrations) is:
```
Comment (mjingle) [inline README.md:25]: Is there a specific file or interface where this configuration should go?

File: README.md [hunk 4]
```diff
@@ -107,31 +98,28 @@ export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
...
 
-6. Test cases also run as part of the pipeline.
+5. Test cases also run as part of the pipeline.
    See [test-terraform-module.yml](https://github.com/honeycombio/terraform-aws-integrations/blob/main/.github/workflows/test-terraform-module.yml)
 
 ### Docs
 
-Docs are autogenerated by running `make generate-docs`, and put
-in [USAGE.md](https://github.com/honeycombio/terraform-aws-integrations/blob/main/USAGE.md).
+Docs located on the Terraform website are autogenerated by running `make generate-docs`, and put in [USAGE.md](https://github.com/honeycombio/terraform-aws-integrations/blob/main/USAGE.md).
```
Comment (mjingle) [inline README.md:111]: Which docs are these? Let's include a link to where the Terraform-hosted docs are. (I assume Terraform ingests this USAGE.md file somehow and places it somewhere in their docs.)

Comment (mjayaram) [reply README.md:111]: https://registry.terraform.io/modules/honeycombio/integrations/aws/latest

Comment (mjayaram) [reply README.md:111]: the registry does not use usage.md though - we have kept this block the same in all our modules FYI

File: README.md (hunk 3 above)
Comment (mjayaram) [reply README.md:25]: given the repo is about terraform modules - do we need to say terraform config? re: file - not really folks can include in their IAC *.tf or add this into the right spot that they will determine

File: README.md (hunk 4 above)
Comment (mjingle) [reply README.md:111]: I did see that the Terraform link displays our current main repo README (nifty!)

Comment (mjingle) [reply README.md:111]: Amended phrasing

**Review by mjayaram — APPROVED** [review]
🎉

**Pull request merged**